### ISSUE TITLE
[2.0.9+1] FAILURE: Build failed with an exception.

### ISSUE THREAD

**Issue by JAICHANGPARK** [florent37/Flutter-AssetsAudioPlayer#312]
**Flutter Version**

My version : 
```
[√] Flutter (Channel master, 1.21.0-6.0.pre.91, on Microsoft Windows [Version 10.0.19041.388], locale ko-KR)
    • Flutter version 1.21.0-6.0.pre.91 at C:\flutter
    • Framework revision aeedd71e84 (41 minutes ago), 2020-07-29 03:41:03 -0400
    • Engine revision a9910e409c
    • Dart version 2.10.0 (build 2.10.0-1.0.dev 24c7666def)


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Users\JEFFREY\AppData\Local\Android\sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: C:\Program Files\Android\Android Studio1\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.
```

**Lib Version**

My version : 2.0.9+1

**Platform (Android / iOS / web) + version**

Platform : Android

**Describe the bug**

```
assets_audio_player-2.0.9+1\android\src\main\kotlin\com\github\florent37\assets_audio_player\playerimplem\PlayerImplemExoPlayer.kt: (152, 118): Unresolved reference: AdtsExtractor

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':assets_audio_player:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 15s
Exception: Gradle task assembleDebug failed with exit code 1

```

**Comment by JAICHANGPARK** [comment]
I tried 
> flutter clean

But I had same build error

**Comment by andrewpmoore** [comment]
I'm seeing exactly the same when building for release mode on android

**Comment by florent37** [comment]
I'm on it 👍

**Comment by florent37** [comment]
thanks to have report this :)

I published a `2.0.9+2` to fix it , it was just a missing import, sorry

**Comment by JAICHANGPARK** [comment]
@florent37 Thank you 🙏

**Comment by nasir447** [comment]
> 
> 
> thanks to have report this :)
> 
> I published a `2.0.9+2` to fix it , it was just a missing import, sorry

it also has same issue
